dist/index.html

```diff
@@ -1,12 +1,164 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x"
+      crossorigin="anonymous"
+    />
+    <title>ToDo List</title>
+  </head>
+
+  <body class="text-white">
+    <header class="mt-5">
+      <h1 class="text-center">TO-DO-LIST APP</h1>
+    </header>
+
+    <main
+      class="
+        container
+        d-flex
+        flex-column
+        align-items-start
+        flex-md-row
+        justify-content-center
+        my-4
+      "
+    >
+      <div class="projects mx-auto mx-md-2 mt-2">
+        <h2 class="text-center mt-3">PROJECTS</h2>
+        <form class="container">
+          <div class="d-flex mb-3">
+            <input
+              placeholder="New Project"
+              type="text"
+              class="form-control m-1"
+              id="new-project"
+            />
+            <button class="new-project-btn btn btn-secondary m-1">
+              <i class="bi bi-plus"></i>
+            </button>
+          </div>
+        </form>
+        <div class="project-items">
+          <ul class="project-items-list fs-4"></ul>
+        </div>
+      </div>
+
+      <div class="todo-list mx-auto mx-md-2 mt-2">
+        <h2 class="text-center mt-3">TODO ITEMS</h2>
+        <button class="add-todo w-100 btn btn-secondary m-1">
+          <i class="bi bi-plus"></i>
+        </button>
+        <div class="todo-items mt-3">
+          <ul class="todo-items-list fs-4"></ul>
+        </div>
+        <div class="buttons d-flex justify-content-evenly my-3">
+          <!-- <button class="btn btn-secondary m-1">Clear Tasks</button>
+        <button class="btn-delete btn btn-secondary m-1">Delete List</button> -->
```

Comment:
Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history. 

![image](https://user-images.githubusercontent.com/31897434/77977676-3d0dec80-72f8-11ea-8176-dc41cb8135b6.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643

Reply:
This comment is not addressed, kindly remove commented code. Use git history instead